README.md

```diff
@@ -19,11 +25,73 @@
 
 `path` - path to `hoppscotch-collection.json` exported from [hoppscotch.io](https://hoppscotch.io/)
 
-## Instructions
+## Detailed Instructions
 
-- Clone the repository.
-- Navigate to the directory and install dependencies with `npm install`.
-- Now fire in `npm link` which creates a symlink and now `hdg` can be accessed globally.
+*Hopp Doc Gen generates documentation (using Vuepress) based on Hoppscotch collections.*
+
+**1.** Navigate to https://hoppscotch.io/doc and import a collection you have previously created in Hoppscotch (by clicking on the folder icon next to any collection in the Collections pane).
+
+**2.** Once you've imported the collection, you'll see some JSON data about the collection, make sure to select all of it and copy it to your clipboard - we're going to be needing it! :wink:
+
+
+**3.** Now it's time to generate the documentation! Clone this repository in the terminal.
+
+```
+$ git clone https://github.com/hoppscotch/hopp-doc-gen.git
+$ cd hopp-doc-gen
+```
+
+**4.** Install all the dependencies.
+
+```
+$ npm install
+```
+
+**5.** Run `npm link` so that we can generate our documentation!.
+
+```
+$ npm link
+```
+
```

Comment:
Instruct the user to install the [hdg npm package](https://www.npmjs.com/package/hdg).

```bash
npm install -g hdg

// or via npx
npx hdg generate [options]
```

Reply:
Wow, thanks! I didn't know there was an NPM package for that!